ur_robot_driver/include/ur_robot_driver/hardware_interface.h

```diff
@@ -55,26 +59,94 @@ namespace ur_robot_driver
  * driver. It contains the read and write methods of the main control loop and registers various ROS
  * topics and services.
  */
-class URHardwareInterface : public hardware_interface::components::SystemInterface
+class URPositionHardwareInterface : public hardware_interface::SystemInterface
 {
 public:
-  RCLCPP_SHARED_PTR_DEFINITIONS(URHardwareInterface);
+  RCLCPP_SHARED_PTR_DEFINITIONS(URPositionHardwareInterface);
+
+  return_type configure(const HardwareInfo& system_info) final;
 
-  hardware_interface::return_type configure(const HardwareInfo& system_info) final;
+  std::vector<hardware_interface::StateInterface> export_state_interfaces() final;
```

Comment:
member functions should be camelCased. e.g. `exportStartInterfaces`, `getStatus`

Reply:
Oh, these were inherited, huh? I guess the naming issue is on the ros2_control side, then

Reply:
Yes , it is inherited

Reply:
Made an issue here we can keep an eye on

https://github.com/ros-controls/ros2_control/issues/269